data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/codec/json/JsonInputCodecConfig.java

```diff
@@ -11,11 +11,13 @@
 package org.opensearch.dataprepper.plugins.codec.json;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.Min;
 import jakarta.validation.constraints.Size;
 
 import java.util.List;
 
 public class JsonInputCodecConfig {
+    static final Integer DEFAULT_MAX_EVENT_LENGTH = 20000000;
```

Comment:
Do you think we should just have this default to null to use the current default, and then just not pass it when it's not set? 

Might be a little safer since it will always use the library default